instance/ansible.py

```diff
@@ -33,11 +33,18 @@
 
 from instance.utils import poll_streams, create_temp_dir
 
+
 # Logging #####################################################################
 
 logger = logging.getLogger(__name__)
 
 
+# Custom ansible plugins ######################################################
+
+ANSIBLE_CALLBACKS_DIR = os.path.join(os.path.dirname(__file__), 'ansible_callbacks')
+ANSIBLE_STDOUT_CALLBACK = 'prettify'
```

Comment:
@mtyaka Rather than using a custom callback, what about using the built in `yaml` callback (see https://www.shellhacks.com/ansible-human-readable-output-format/). That might work better for web display (fewer double quotes) and would mean less code for us to maintain. Although that would be a major rework to what you have done. Maybe a follow up ticket.

Reply:
I tried the built-in `yaml` (and `json`) callbacks before creating a custom one. The `yaml` callback is unfortunately unusable for our purpose. Its output is very similar to the default output and not much easier to pretty print.
The `json` callback is even less useful because it buffers the output of the entire playbook and only outputs a huge dump of json when the playbook finishes running.